src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -2,34 +2,121 @@
  * @file Redesigned Metadata sidebar component
  * @author Box
  */
-import * as React from 'react';
+import { MetadataEmptyState } from '@box/metadata-editor';
 import flow from 'lodash/flow';
+import getProp from 'lodash/get';
+import * as React from 'react';
+import type { MessageDescriptor } from 'react-intl';
 import { FormattedMessage } from 'react-intl';
+import type { ElementsXhrError } from '../../common/types/api';
+import type { BoxItem } from '../../common/types/core';
+import type { MetadataEditor, MetadataTemplate } from '../../common/types/metadata';
+import {
+    FIELD_IS_EXTERNALLY_OWNED,
+    FIELD_PERMISSIONS,
+    FIELD_PERMISSIONS_CAN_UPLOAD,
+    IS_ERROR_DISPLAYED,
+    ORIGIN_METADATA_SIDEBAR_REDESIGN,
+} from '../../constants';
+import { normalizeTemplates } from '../../features/metadata-instance-editor/metadataUtil';
+import { isUserCorrectableError } from '../../utils/error';
```

Comment:
did you receive any lint issues with the order of these imports?

Reply:
no issues, should I sort them in specific order?

Reply:
refer to this https://github.com/box/box-ui-elements/blob/master/src/elements/content-preview/ContentPreview.js on the sort order